src/app/routes/utils/fetchPageData/index.js

```diff
@@ -32,22 +32,39 @@ export const getUrl = pathname => {
   return `${baseUrl}${basePath.replace(ampRegex, '')}.json${params}`; // Remove .amp at the end of pathnames for AMP pages.
 };
 
+const validateData = data => {
+  if (data) {
+    return 'content' in data;
```

Comment:
I've never seen this syntax before - what does it do (if you don't mind me asking?)

Reply:
I do mind you asking that's very private 😆 joking. So it checks for the `content` key in the object `data`. True if it ha one and false if not

Reply:
thank you for explaining!